package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tram-one",
-  "version": "8.1.3",
+  "version": "9.0.0-beta.2",
```

Comment:
is it okay to have the beta version in a master branch? or is the versioning managed by npm?

Reply:
When we do the final merge, it'll be just 9.0.0